protocol/scripts/genesis/sample_pregenesis.json

```diff
@@ -0,0 +1,1723 @@
+{
```

Comment:
Is this file the result of running prod_pregenesis.sh? Checking it in isn't that useful if it can go out of sync.

Reply:
Agreed. In the interest of time, I'll manually keep them in sync for this PR. Can work on a script later to validate